docs/source/installation/plugins.rst

```diff
@@ -132,8 +132,8 @@ Manim will look for ``__all__`` keyword defined in ``manim_plugintemplate`` and
 everything as a global variable one by one.
 
 If ``object_reference`` is a function, Manim calls the function and expects the
-function returns a list of modules or functions that needs to defined globally and
-it defined it.
+function returns a list of modules or functions that needs to be defined globally and
+in defined it.
```

Comment:
I think the whole sentence is badly written and changing this single word doesn't really fix it. Can you rephrase the whole sentence?